### ISSUE TITLE
responsiveSlides activated + autoScrolling:true 

### ISSUE THREAD

**Issue by delacrouz** [alvarotrigo/fullPage.js#3147]
Hi!

I just bought the responsiveSlides extension of fullpage. 
In fact, we have horizontal slides that we would like to be vertical under 992px. 

So we set the responsive Width to 992px. However, we just want the "responsiveSlides" feature and we don't want to break the autoscrolling (we want autoscrolling true under 992px).

How can I do this?

Here is my code : 
            responsiveSlides: true,
            responsiveWidth:992,
            responsiveHeight:500,
            autoScrolling:true,

Thank you!

**Comment by alvarotrigo** [comment]
You can try using the [afterResponsive callback](https://github.com/alvarotrigo/fullPage.js#afterresponsiveisresponsive) for that.

```javascript
afterResponsive = function(isResponsive){
     if(isResponsive){
         $.fn.fullpage.setAutoScrolling(true);
     }
}
```

Let me know if that works for you.

**Comment by delacrouz** [comment]
Hi @alvarotrigo it's perfect problem solved ;-)

Thank you!

Antoine

**Comment by delacrouz** [comment]
Hi @alvarotrigo,

Finally, I had an other problem with this extension.
On my webpage, where I use fullpage, I have 5 sections.

The section 1 to 3 are normal sections with not too much content..
The section 4 is a slider with 4 slides.
The section 5 is a large section, where I have a scrolling bar on desktop to scroll this large section. 

So it's working perfectly on desktop, but on mobile, because of the snippet of code I added, we cannot scroll anymore in this section.

```
afterResponsive: function(isResponsive){
     if(isResponsive){
         $.fn.fullpage.setAutoScrolling(true);
     }
}
```
I tried to add a "overflow: scroll;" in CSS to my section under 992px but it't nos working.
I can send you the link in private if you want!

Thank you!

**Comment by alvarotrigo** [comment]
> So it's working perfectly on desktop, but on mobile, because of the snippet of code I added, we cannot scroll anymore in this section.

You'll have to use `scrollOverflow: true`

**Comment by delacrouz** [comment]
I already use it but it's not working on MOBILE only. 

`$('.page-template-about-us #fullpage').fullpage({
            anchors: ['explore', 'made-for-your-market', 'save-yourself', 'services', 'leadership'],
            menu: '#primary-menu',
            scrollHorizontallyKey: '#',
            scrollHorizontally: true,
            scrollOverflow: true,
            resetSlidersKey: '#',
            resetSliders: true,
            responsiveSlidesKey: '#',
            responsiveSlides: true,
            responsiveWidth:992,
            responsiveHeight:500,
            afterResponsive: function(isResponsive){
            if(isResponsive) {
                $.fn.fullpage.setAutoScrolling(true);
                }
                
                
            }
            
	});`

Thank you!

**Comment by alvarotrigo** [comment]
Oh right... that's because scrolloverflow get's automatically destroyed on responsive... 
That's how it is meant to work :)

You'll have to trick fullPage.js. Try this:

```javascript
afterResponsive: function(isResponsive){
    if(isResponsive) {
        $.fn.fullpage.setAutoScrolling(true);
        $('body').removeClass('fp-responsive');
        $.fn.fullpage.reBuild();
        $('body').addClass('fp-responsive');
    }
}
```

**Comment by delacrouz** [comment]
Hey ! Thanks for your reactivity. It's not working however.

When I remove the "responsiveSlides" extension, it's working both on mobile and desktop. The issue seems to be related to this extension, and how the slides are converted to vertical sections.

` responsiveSlides: true,
responsiveWidth:992,
responsiveHeight:500,
`

**Comment by delacrouz** [comment]
Also, when I refresh my page under 992px, it's not working. However, when I just "resize" the screen, the scroll is working, so I think it's related to the code we put in the afterResponsive part.

**Comment by alvarotrigo** [comment]
Oh right...  It only gets removed when using `fp-auto-height-responsive` :) My bad.
Make sure to use the latest [version 0.0.7 in the non minified file of the dev branch](https://github.com/alvarotrigo/fullPage.js/blob/dev/vendors/scrolloverflow.js).

I am not able to replicate your issue.

I'm using:

```javascript
			afterResponsive: function(isResponsive){
			    if(isResponsive) {
			        $.fn.fullpage.setAutoScrolling(true);
			    }
			},
```

And it works just fine. 
If you still issues please try [contact me](https://alvarotrigo.com/#contact) with a link to an isolated  reproduction and remove any CSS or JS files or code external to fullPage.js that is not strictly necessary for the reproduction. 
Use empty sections or only `<h1>`  headers to create the scrollable elements.

**Comment by delacrouz** [comment]
I tried with the last file, it's not working also. 

I have an idea, would it be possible to have ResponsiveSlides work under 992px without using the "responsiveWidth" option? 

Because I just want to have vertical sections under 992 for my slider, but I don't other thing to change. Thank you!

**Comment by delacrouz** [comment]
Hey ! 

So I ended up with this solution, using your link (https://alvarotrigo.com/fullPage/extensions/responsive-slides.html) with the "fire to section".

So I removed the "responsiveWidth:992" and the "responsiveSlides:true" of my options, and I added in my javascript : 

`if ($(window).width() < 992) {
        $.fn.fullpage.responsiveSlides.toSections();
      }`

So now, I have my slides converted under 992px and I have the scroll still working like on desktop on mobile. And it works!
